uvicorn/config.py

```diff
@@ -121,6 +121,7 @@ def __init__(
         self.ssl_ciphers = ssl_ciphers
 
         self.loaded = False
+        self.loop_setup = False
```

Comment:
Okay, last bit. Let's remove this. It's safe to recall `Config.setup_loop()` and in any case it's *only* called from the `Server` class and isn't expected to be called elsewhere. So... We can just drop this bit of state.